src/main/java/lotto/dto/LottoNumbers.java

```diff
@@ -0,0 +1,27 @@
+package lotto.dto;
+
+import java.util.List;
+
+public class LottoNumbers {
+
+    private final List<LottoNumber> lottoNumbers;
+
+    public LottoNumbers(List<LottoNumber> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    public List<LottoNumber> getLottoNumbers() {
+        return lottoNumbers;
+    }
+
+    public boolean isContain(int bonusNumber) {
+        return lottoNumbers.stream()
+                .anyMatch(lottoNumber -> bonusNumber == lottoNumber.getLottoNumber());
+    }
```

Comment:
작성해주선 코드를 확인해보면 stream미사용시 for문안에 if문이 포함되어 있기 때문에 2depth가 됩니다.
프로그래밍 요구사항을 다시 확인해주세요 :)

```suggestion
     public boolean isContain(int bonusNumber) {
        for (LottoNumber lottoNumber : lottoNumbers) {
            if (bonusNumber == lottoNumber.getLottoNumber()) {
                return true;
            }
        }
        return false;
    }
```

Reply:
step2에서 받은 피드백인데 이번에도 똑같은 실수를 한 것 같아서 죄송합니다... 피드백 반영했습니다!!